recipe-server/client/control_new/components/extensions/CreateExtensionPage.js

```diff
@@ -0,0 +1,57 @@
+import { message } from 'antd';
+import autobind from 'autobind-decorator';
+import React, { PropTypes as pt } from 'react';
+import { connect } from 'react-redux';
+import { push } from 'redux-little-router';
+
+import ExtensionForm from 'control_new/components/extensions/ExtensionForm';
+import { createExtension } from 'control_new/state/extensions/actions';
+
+
+@autobind
+export class _CreateExtensionPage extends React.Component {
```

Comment:
Fair points. I'd go with `Unwrapped`/`Unconnected` over the underscore, but I'm not leaning towards one or the other. Up to you.